index.js

```diff
@@ -4,7 +4,11 @@ const execa = require('execa');
 
 const handler = err => {
 	if (err.code === 'ENOENT') {
-		throw new Error('Couldn\'t find the required `xsel` binary. On Debian/Ubuntu you can install it with: sudo apt install xsel');
+		if (err.path === 'termux-clipboard-get' || err.path === 'termux-clipboard-set') {
```

Comment:
I would use `let message = 'default message';` and change it in an if statement so you only have one `throw`.